src/Core/Content/Product/Cms/ProductDescriptionReviewsCmsElementResolver.php

```diff
@@ -56,12 +58,13 @@ public function enrich(CmsSlotEntity $slot, ResolverContext $resolverContext, El
             $product = $this->getSlotProduct($slot, $result, $productConfig->getStringValue());
         }
 
-        if ($product instanceof SalesChannelProductEntity) {
+        $data->setProduct($product);
+
+        if ($product instanceof SalesChannelProductEntity && $this->systemConfigService->getBool('core.listing.showReview')) {
```

Comment:
now, we check the config twice :thinking: is it better performing if you catch the exception around the `load` method?

Reply:
I'd argue this has a better performance and is more readable since the system config is cached anyway and if showReview is false we don't want to load it